### DISCUSSION TITLE
"Remove dialogue dashes from single lines" not quite working

### DISCUSSION THREAD

**Discussion by DrReddP** [SubtitleEdit/subtitleedit#5087] [General]
How do I get it to remove dialogue dashes from lines with on one dash?
Image 1 - common errors settings
Image 2 - dialogue dashes it leaves behind
![image](https://user-images.githubusercontent.com/82276985/120506769-3c596600-c394-11eb-8cd8-c0edf55195ad.png)
![image](https://user-images.githubusercontent.com/82276985/120506812-42e7dd80-c394-11eb-9239-f00b42a2fc95.png)

**Comment by niksedk** [comment]
OK, got the same problem...

How is this?
![image](https://user-images.githubusercontent.com/1261331/120891466-1a1c4e00-c609-11eb-980b-b5c1c0661f42.png)

https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.1/SubtitleEditBeta.zip

The old "Remove dialog dashes in single lines" should probably be removed...

**Comment by OmrSi** [comment]
Should also close https://github.com/SubtitleEdit/subtitleedit/issues/4441
